src/elements/Pagination/Readme.md

```diff
@@ -0,0 +1,27 @@
+### Basic Pagination
+```jsx
+  const currentPage = 11;
+  const itemsPerPage = 10;
+  const totalItems = 124;
+  const onChange = (page) => {};
+
+  <Pagination currentPage={currentPage} itemsPerPage={itemsPerPage} totalItems={totalItems} onChange={onChange} />
```

Comment:
you can just do
```
<Pagination currentPage={11} itemsPerPage={10} totalItems={124} onChange={() => {} />
```

Reply:
As the source code will be shown in style guide, I want to make is more clear to read.

Reply:
IMO, `currentPage={11}` is clearer than `currentPage={currentPage}`.
Also, this is documentation what data type they need to use for each prop, using original data type into the prop is better to understand.
This is how material-ui is doing.
<img width="977" alt="screen shot 2017-09-19 at 4 14 30 pm" src="https://user-images.githubusercontent.com/1316039/30619744-f89dc220-9d55-11e7-82e7-a2cd25fb0fd3.png">